vocab/credentials/credentials.ttl

```diff
@@ -19,6 +19,14 @@ cred:JsonSchemaValidator2018 a rdfs:Class;
   rdfs:label ""@en;
   rdfs:comment """A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language."""@en;
   rdfs:isDefinedBy cred: .
+cred:JsonSchema2023 a rdfs:Class;
+  rdfs:label ""@en;
+  rdfs:comment """A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language."""@en;
+  rdfs:isDefinedBy cred: .
+cred:CredentialSchema2023 a rdfs:Class;
+  rdfs:label ""@en;
+  rdfs:comment """A type of validator that can be used to syntactically validate JSON documents packaged as Verifiable Credentials using the JSON Schema language."""@en;
+  rdfs:isDefinedBy cred: .
```

Comment:
```suggestion
```

This is modifying the v1 Turtle vocabulary, which we shouldn't be doing for a v2 feature.